modules/nf-core/iqtree/tests/main.nf.test.snap

```diff
@@ -17,21 +17,21 @@
                 ""
             ],
             "5",
-            "IQ-TREE 2.3.4 COVID-edition built May 23 2024",
-            "IQ-TREE multicore version 2.3.4 COVID-edition for Linux x86 64-bit built May 23 2024",
+            "IQ-TREE 2.4.0 built Feb 12 2025",
+            "IQ-TREE multicore version 2.4.0 for Linux x86 64-bit built Feb 12 2025",
```

Comment:
https://nf-co.re/docs/guidelines/components/modules#emission-of-versions

All reported versions must start with a numeric character. So I would remove everything except for: "2.4.0".

Reply:
Are you talking about whats in the `versions.yml`? That is already just `2.4.0`. I'm not sure where that info in the snap is coming from, perhaps standard out/error.

Reply:
Indeed, you are right!